### PR TITLE
Define firmware version minor for payload in version.h

### LINKED ISSUE #28: 1.6.7 reports version as 1.6.4
The firmware from https://www.dragino.com/downloads/index.php?dir=LGT_92/firmware/v1.6.7/ reports its version as 1.6.4 in the LoRaWAN message.

The decoder/payload formatter is defined as using `var Firmware = 160 + (bytes[10] & 0x1f)` to decode the version. With byte 11 being 0x64 (e.g. as in `02 D4 3C 64 00 88 A1 00 4D DC 64`) this means 160 + 4 => 1.6.4. instead of 1.6.7.

https://github.com/dragino/LGT-92_-LoRa_GPS_Tracker/blob/4800dc2451745dd020654f8391c94a405ac24eac/Projects/Multi/Applications/LoRa/DRAGINO-LRWAN(AT)/src/lora.c#L122

has the correct version. So, I don't really understand what's going wrong here.

The version reported on UART is correct

```
LSI=37024
LGT-92 Device
Image Version: v1.6.7
LoRaWan Stack: DR-LWS-005
```

### PR DESCRIPTION
**Pull request by tormodvolden**
Fixes #28

### CODE REVIEW

**Comment by marcelstoer** [comment]
Thanks for this fix!

To improve things even more and to reduce the risk of someone forgetting to bump the version in all the required places maybe you could define:
- `FIRMWARE_VERSION_MAJOR`, `FIRMWARE_VERSION_MINOR`, and `FIRMWARE_VERSION_PATCH` in `version.h`
- make `#define AT_VERSION_STRING` in `version.h` use these three defines (rather than re-defining it)
- use these defines for https://github.com/dragino/LGT-92_-LoRa_GPS_Tracker/blob/master/Projects/Multi/Applications/LoRa/DRAGINO-LRWAN(AT)/src/lora.c#L122 as well
- use these defines in all the other places I forgot (i.e. didn't find)...

File: Projects/Multi/Applications/LoRa/DRAGINO-LRWAN(AT)/inc/version.h [hunk 1]
```diff
@@ -73,6 +73,7 @@ Maintainer: Miguel Luis, Gregory Cristian and Wael Guibene
 #define LRWAN_VERSION  (uint32_t) 0x00001140  /*3 next hex is i_cube release*/
 #define VERSION   (uint32_t) ( LORA_MAC_VERSION | LRWAN_VERSION | TEST_VERSION )
 #define AT_VERSION_STRING	"v1.6.7"
+#define FIRMWARE_VERSION_MINOR	7
```
Comment (marcelstoer) [inline Projects/Multi/Applications/LoRa/DRAGINO-LRWAN(AT)/inc/version.h:76]: It's actually not the minor version but the _patch_ version.

Comment (tormodvolden) [reply Projects/Multi/Applications/LoRa/DRAGINO-LRWAN(AT)/inc/version.h:76]: true dat! I will update the patch.

Your other suggestions make sense too, they are a little more work but better in the long term.

**Comment by marcelstoer** [comment]
@tormodvolden did you find instructions somewhere how to build this thing? I am a little surprised they don't have a CI process hooked up to this repository.

**Comment by tormodvolden** [comment]
@marcelstoer It is not so surprising given that it has been built with a proprietary GUI IDE (I don't know the terms of use and details, I never used it), and this github repository is not where it is being developed and maintained, but more a place to publish it.

Build instructions can be found in the [user manual](https://www.dragino.com/downloads/index.php?dir=LGT_92/), chapter 5 "Developer Guide" and seem well described.

**Comment by tormodvolden** [comment]
About the GCC support: We should add a README with a big fat warning that if you simply flash your LGT-92 with the result of the build, it will wipe out the flash settings with LORA IDs and keys etc. Maybe the Makefile flash target should check that there are flash settings at the new, higher location before proceeding to flash the firmware.

**Pull request merged**